lib/collectionspace/converter/materials/config.yml

```diff
@@ -0,0 +1,93 @@
+---
+registered_authorities:
+  - orgauthorities
+  - personauthorities
+  - placeauthorities
+  - conceptauthorities
+  - workauthorities
+  - citationauthorities
+  - nameauthorities
+  - subjectauthorities
+registered_procedures:
+  - CollectionObject
+  - Material
+  - Media
+  - Person
+registered_profiles:
+  cataloging:
+    Procedures:
+      CollectionObject::
```

Comment:
@Paulmulonzia there's a trailing ":" character here. Believe this is the cause of "wrong constant name MaterialsCollectionObject:".